cfme/tests/infrastructure/test_individual_host_creds.py

```diff
@@ -33,6 +33,9 @@ def get_host_data_by_name(provider_key, host_name):
 
 
 # Tests to automate BZ 1278904
+@pytest.mark.meta(blockers=[BZ(1516849,
+                            forced_streams=['5.8', '5.9', 'upstream'],
+                            unblock=lambda provider: provider.type != 'rhevm')])
```

Comment:
**Required**

Do not directly compare type strings, use the provider classes and the `provider.one_of()` method - lots of examples throughout the code base.